pyface/ui/wx/timer/do_later.py

```diff
@@ -13,4 +13,4 @@
 Provided for backward compatibility.
 """
 
-from pyface.timer.do_later import DoLaterTimer
+from pyface.timer.do_later import DoLaterTimer  # noqa: F401
```

Comment:
`DeprecationWarning`?

Reply:
i'll include this with the issue for the `do_later` module on qt.

Reply:
resolving this too because i opened https://github.com/enthought/pyface/issues/916